converter/fluka/cards/scoring_card.py

```diff
@@ -1,42 +1,112 @@
 from dataclasses import dataclass, field
+from typing import Optional, Union
 
 from converter.fluka.cards.card import Card
-from converter.fluka.helper_parsers.scoring_parser import Scoring
+from converter.fluka.helper_parsers.detector_parser import Detector
+from converter.fluka.helper_parsers.scoring_parser import CustomFilter, ParticleFilter, Quantity, Scoring
 
 
-def handle_first_card(scoring: Scoring, output_unit: int = 21) -> str:
-    """Creates first Scoring card"""
+@dataclass
+class ScoringCardIndexCounter:
+    """Class representing counter for Fluka scoring cards"""
+
+    usrbin_counter: int = 0
+
+
+def handle_scoring_cards(output_unit: int, scoring: Scoring, counter: ScoringCardIndexCounter) -> str:
+    """Creates Scoring cards"""
+    output: list[str] = []
+    for quantity in scoring.quantities:
+        usrbin_card = handle_usrbin_scoring(scoring.detector, quantity, output_unit, counter)
+        output.append(usrbin_card)
+    return '\n'.join(output).strip()
+
+
+def short_name(name: str) -> str:
+    """Creates short name for fluka card"""
+    return f'{name[:10]}'
```

Comment:
```suggestion
    return name[:10]
```

Add a comment explaining why only 10 characters are taken.